sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/manager/NoKeyKeyManager.java

```diff
@@ -0,0 +1,29 @@
+package org.sdase.commons.keymgmt.manager;
+
+import java.util.Collections;
+import java.util.Locale;
+import java.util.Set;
+
+public class NoKeyKeyManager implements KeyManager {
+
+  private final String keyDefinitionName;
+
+  public NoKeyKeyManager(String keyDefinitionName) {
+    this.keyDefinitionName = keyDefinitionName;
+  }
+
+  @Override
+  public String getKeyDefinitionName() {
+    return keyDefinitionName.toUpperCase(Locale.ROOT);
+  }
+
+  @Override
+  public Set<String> getValidValues() {
+    return Collections.emptySet();
+  }
+
+  @Override
+  public boolean isValidValue(String testable) {
+    return false;
```

Comment:
No sure if "false" makes more sense here than "true" as default value. Might be helpful to get some documentation for the contract to decide.

Reply:
add some documentation to NoKeyManager